### PR TITLE
[Improvement][E2E]e2e improve  add workflow httpTask e2e case

### PR DESCRIPTION
**Pull request by jackyyyyyssss**
## Purpose of the pull request
E2e improve  add workflow httpTask e2e case

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

**Comment by jackyyyyyssss** [comment]
@SbloodyS  The code modified this time will not affect the backend in the e2e package. It seems that the error is due to a port conflict on the GitHub machine. I tried again in the local repository and it was successful. The use case for this time has been successful. Please help me try again
image
image
image

**Review by songjianet — DISMISSED** [review]

File: dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/resources/docker/workflow-http/mockserver-config.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      {
+        "name": "pizz2",
+        "age": 19
+      }
+    ],
+    "headers": {
+      "Content-Type": "application/json"
+    }
+  }
+}]
```
Comment (SbloodyS) [inline dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/resources/docker/workflow-http/mockserver-config.json]: The last line of each file should be blank.

**Review by SbloodyS — DISMISSED** [review]
Generally LGTM. Just some NIT.

File: dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/resources/docker/workflow-http/mockserver-config.json (hunk 1 above)
Comment (jackyyyyyssss) [reply dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/resources/docker/workflow-http/mockserver-config.json]: done PTAL @SbloodyS  thks

**Review by SbloodyS — APPROVED** [review]

**Pull request merged**